app/test_main.py

```diff
@@ -1,3 +1,25 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_human_age_zero() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_human_age_in_1_year() -> None:
+    assert get_human_age(15, 15) == [1, 1]
+
+
+def test_human_age_in_2_year() -> None:
+    assert get_human_age(24, 24) == [2, 2]
+
+
+def test_difference_between_cat_dog_year() -> None:
+    assert get_human_age(28, 28) == [3, 2]
+
+
+def test_human_age_in_3_year() -> None:
+    assert get_human_age(28, 29) == [3, 3]
+
+
+def test_large_ages() -> None:
+    assert get_human_age(100, 100) == [21, 17]
```

Comment:
To ensure comprehensive test coverage, consider adding tests for edge cases such as negative numbers, zero, or very large numbers, and check if the function raises the correct exception for incorrect data types, as suggested in the checklist.